store/errors.go

```diff
@@ -182,9 +182,6 @@ var (
 func translateSnapActionError(action, code, message string) error {
 	switch code {
 	case "revision-not-found":
-		if action == "refresh" {
-			return ErrNoUpdateAvailable
```

Comment:
I assume this is just not needed anymore, right? I.e. we still return ErrNoUpdateAvailable at the right place.

Reply:
correct, I moved the backward compat logic all in daemon